olive/data/component/post_process_data.py

```diff
@@ -8,8 +8,20 @@
 from olive.data.registry import Registry
 
 
+@Registry.register_post_process()
```

Comment:
is there a reason it is registered using this also?

Reply:
With default registration alone, it can't explicitly be used in json configs. A default component type is still a legal component type to be used in config and this registration allows that.